src/hera/workflows/v5/examples/upstream/upstream.dag-diamond.yaml

```diff
@@ -0,0 +1,44 @@
+# The following workflow executes a diamond workflow
```

Comment:
This is synced automatically from upstream.

Convention is examples/upstream.<example>.yaml <> <example>.py